tensorflow/tools/ci_build/Dockerfile.rocm

```diff
@@ -67,7 +69,7 @@ RUN apt-get install -y software-properties-common
 # Install rocm pkgs
 RUN apt-get update --allow-insecure-repositories && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
-    rocm-dev rocm-libs rccl && \
+    rocm-dev rocm-libs rccl miopengemm && \
```

Comment:
Why is this needed? TF doesn't use miopengemm.

Reply:
removed miopengemm in new patchset